components/core/DataMeter.js

```diff
@@ -115,10 +158,88 @@ export const DataMeter = (props) => {
         {Strings.bytesToSize(props.stats.bytes)} of {Strings.bytesToSize(props.stats.maximumBytes)}{" "}
         used
       </div>
-      <DataMeterBar bytes={props.stats.bytes} maximumBytes={props.stats.maximumBytes} />
+
+      <div style={{ position: "relative" }}>
+        <div css={STYLES_DATA_METER_BASE}></div>
+        <DataMeterBar
```

Comment:
I think the strategy should be inline-flex elements like the codepen I posted earlier: https://codepen.io/meanjim/pen/pobKKbr